ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -35,17 +35,23 @@ cp "${SHARED_DIR}/install-config.yaml" "${dir}/"
 mkdir -p ~/.ssh
 cp "${SSH_PRIV_KEY_PATH}" ~/.ssh/
 
-TF_LOG=debug openshift-install --dir="${dir}" create cluster 2>&1 | grep --line-buffered -v password &
+TF_LOG=debug openshift-install --dir="${dir}" create cluster 2>/dev/null
 
 set +e
 wait "$!"
-ret=$?
+ret="$?"
 set -e
 
+if [[ -f "${dir}/.openshift_install.log" ]]; then
+  sed 's/password: .*/password: REDACTED/' "${dir}/.openshift_install.log" >"${ARTIFACT_DIR}/.openshift_install.log"
+fi
+
+if [[ "${ret}" -ne 0 ]; then
+  cp "${dir}"/log-bundle-*.tar.gz "${ARTIFACT_DIR}/" 2>/dev/null || :
+fi
```

Comment:
we create this if condition when we are skipping the error condition already ??

Reply:
> we create this if condition when we are skipping the error condition already ??

Yeah, this just avoids noise like:

```console
$ cp does-not-exist-* /tmp/
cp: cannot stat 'does-not-exist-*': No such file or directory
```

in some of the cases where we know we won't have a log bundle.

Reply:
wouldn't that already be covered with `2>/de/null`

Reply:
Dropped the conditional with 1044cf8d1 -> 1a7b314f9.